### ISSUE TITLE
Hunspell won't work in Flatpak Flathub LibreOffice

### ISSUE THREAD

**Issue by aldevel** [flathub/org.libreoffice.LibreOffice#62]
KDE neon 5.14
KDE Plasma version: 5.14.4
KDE Frameworks Version: 5.52.0
Qt Version: 5.11.2
Kernel Version: 4.15.0-42-generic
OS Type: 64-bit
Flatpak version: 1.0.1
___
Flatpak_-_Flathub_-_LibreOffice version:
Version: 6.1.3.2
Build ID: 86daf60bf00efa86ad547e59e09d6bb77c699acb
CPU threads: 8; OS: Linux 4.15; UI render: default; VCL: gtk3; 
Flatpak
Locale: en-US (en_US.UTF-8); Calc: group threaded
___
I'm looking for Hungarian spell checking and I don't know exactly how language correction works in Flatpak's LibreOffice however what I know the dpkg version of LibreOffice is able to correct Hungarian language through Hunspell package. Other extensions and implemented languages by default won't work in Flatpak nor dpkg version of LibreOffice. The single working solution I found only for Hungarian language the obviously unsupported "Hungarian Dictionary Pack" from the also discounted OpenOffice page: https://extensions.openoffice.org/en/project/hu_dicts
Anyways, I think Hunspell should have been implemented into Flatpak's LibreOffice or make it compatible to work with the dpkg version of Hunspell somehow. Please fix this issue or let me know if I missed something and how to solve it.
Thanks!

**Comment by stbergmann** [comment]
You maybe have the underlying org.freedesktop.Platform.Locale//1.6 only partially installed (or not at all), so that Hungarian locale data (including hunspell files) is missing from it.  To find out, `flatpak list --all` and see if the "org.freedesktop.Platform.Locale/.../1.6" (if present at all) mentions "partial".  To install fully, `flatpak install --reinstall flathub org.freedesktop.Platform.Locale//1.6` (or whatever the flathub remote is called for you instead of "flathub").

**Comment by aldevel** [comment]
Thanks you so much for the very quick response and for your help it works great, awesome. Based on your comment here https://github.com/flatpak/flatpak/issues/702, I already tried this however when I learned that it wasn't installed in my Flatpak installation by default and because of other comments in above thread and also I didn't find any reasonable guide about this particular remote, I just forget it.
Anyways there are a lot available languages and Hungarian spell checking works great now. Thanks again for your help!
